velox/connectors/hive/storage_adapters/s3fs/S3Config.h

```diff
@@ -108,12 +109,14 @@ class S3Config {
              std::make_pair("socket-timeout", std::nullopt)},
             {Keys::kMaxConnections,
              std::make_pair("max-connections", std::nullopt)},
-            {Keys::kMaxAttempts, std::make_pair("max-attempts", std::nullopt)},
-            {Keys::kRetryMode, std::make_pair("retry-mode", std::nullopt)},
+            {Keys::kMaxAttempts, std::make_pair("max-attempts", "10")},
```

Comment:
Let's keep the library defaults since they probably suit the average case. These can be set in the deployment.